EOLib.Localization/EOResourceID.cs

```diff
@@ -152,8 +152,13 @@ public enum EOResourceID
         DIALOG_SHOP_CRAFT_MISSING_INGREDIENTS = 158,
         DIALOG_SHOP_CRAFT_PUT_INGREDIENTS_TOGETHER = 159,
 
+        DIALOG_BARBER_CHANGE_MODAL = 160,
+        DIALOG_BARBER_CHANGE_HAIR_COLOR = 161,
+        DIALOG_BARBER_BUY_HAIRSTYLE = 162,
         DIALOG_WORD_CURRENT = 163,
-
+        DIALOG_BARBER_DO_YOU_WANT_TO_BUY_A_NEW_HAIRSTYLE = 164,
+        
+        DIALOG_TRADE_PLAYERS_NEED_TO_TRADE_ONE_ITEM = 165,
```

Comment:
This new line is a duplicate of the resource below it and doesn't appear to be used in this PR? I think this line can be deleted